core/config/src/main/java/org/apache/causeway/core/config/CausewayConfiguration.java

```diff
@@ -115,6 +113,14 @@ public CausewayConfiguration(final ConfigurableEnvironment environment) {
     private final Security security = new Security();
     @Data
     public static class Security {
+
+        /**
+         * If set, allows <i>Actions</i> with SAFE Semantics to be invoked with only VIEWING permissions.
```

Comment:
trivial : @link to `@Action` annotation and `Semantics` enum?

Reply:
yup, good point